cdap-source-control/src/main/java/io/cdap/cdap/sourcecontrol/operationrunner/InMemorySourceControlOperationRunner.java

```diff
@@ -66,16 +71,17 @@ public class InMemorySourceControlOperationRunner extends
   private final RepositoryManagerFactory repoManagerFactory;
 
   @Inject
-  InMemorySourceControlOperationRunner(RepositoryManagerFactory repoManagerFactory) {
+  public InMemorySourceControlOperationRunner(RepositoryManagerFactory repoManagerFactory) {
```

Comment:
injected constructors should almost never be public to prevent manual constructor outside of unit tests.

Reply:
This was needed to construct `InMemorySourceControlOperationRunner` in `PullOperation` test. I will try to find a better way

Reply:
Fixed